demo/fiber/oneshot_test/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does Loom provide the copyright note for this file?

Reply:
I didn't find any copyright note to this file in loom. May I add the copyright note https://github.com/Tencent/TencentKona-8/blob/KonaFiber/demo/fiber/continuation_launch_test/pom.xml here to this pom file?

Reply:
> May I add the copyright note https://github.com/Tencent/TencentKona-8/blob/KonaFiber/demo/fiber/continuation_launch_test/pom.xml here to this pom file?

No, please just keep it empty.